tests/e2e/test_rosacli_idp.go

```diff
@@ -680,4 +680,149 @@ var _ = Describe("Edit IDP",
 				Expect(textData).Should(ContainSubstring("Failed to create IDP for cluster '%s': Expected a valid certificate bundle: open %s: no such file or directory", clusterID, invalidCaFilePath))
 			})
 
+		It("can Create multiple HTPasswd IDPs including creating admin flow by the rosacli command - [id:65160]",
+			labels.Medium, labels.Runtime.Day2, labels.Feature.IDP,
```

Comment:
Check if `labels.Feature.IDP` is label in Describe level. No need to label in spec level.

Reply:
Yes `labels.Feature.IDP` is available at describe level, removed it from spec level, Thanks!